Resources/Maps/stationstation.yml

```diff
@@ -1152,13 +1152,6 @@ entities:
     pos: 0.5223687,7.507263
     rot: -1.5707963267949 rad
     type: Transform
-- type: HelmetSecurity
```

Comment:
Just out of curiosity, did you modify this file by hand? Any changes done to the map should be done in-game and then saved with a command.

Reply:
It's actually not a problem anymore to edit the map by hand anymore, because I made the UIDs explicit.

Obviously for large changes it's not wieldy. but for small changes like this it's fine now.